apps/emqx_node_rebalance/src/emqx_node_rebalance_api.erl

```diff
@@ -248,10 +249,10 @@ schema("/load_rebalance/:node/evacuation/stop") ->
     }}.
 
 '/load_rebalance/availability_check'(get, #{}) ->
-    case emqx_node_rebalance_status:local_status() of
-        disabled ->
+    case emqx_node_rebalance_status:availability_status() of
```

Comment:
This method was introduced to make the `availability_status` calculation just a persistent term lookup.

This also required some refactoring to make `emqx_eviction_agent:enable_status/0` source of truth.
The originally intended behavior was also implemented: while rebalance/evacuation are in `waith_health_check` state, we report the node to be unavailable but do not forbid connections.